### ISSUE TITLE
show neural net overfitting by hand

### ISSUE THREAD

**Issue by ajschumacher** [ajschumacher/ajschumacher.github.io#156]
pippin overfit_by_hand; ~/aaron/bad_minima bad_minima env

in write-up, ref: https://arxiv.org/abs/1704.08045 "The loss surface of deep and wide neural networks"

---

(not sure about this stuff, but I had a note, so here it is...)

contact guy after weird example is up?

http://www.keerthis.com/Optimization_and_Generalization_Keerthi_Criteo_November_08_2017.pptx

(found that on https://blog.acolyer.org/2017/11/24/on-the-information-bottleneck-theory-of-deep-learning/)

In the end phase, sgd moves within a region of low loss values and settles in regions with good flatness properties

All global minima have flat regions. (slide 31)

Show him https://papers.nips.cc/paper/6770-train-longer-generalize-better-closing-the-generalization-gap-in-large-batch-training-of-neural-networks and also my degenerate global minimum?
